pkgs/applications/networking/wg-bond/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, lib, rustPlatform, fetchFromGitLab, wireguard-tools, makeWrapper }:
+rustPlatform.buildRustPackage rec {
+  pname = "wg-bond";
+  version = "0.2.0";
+  src = fetchFromGitLab {
```

Comment:
```suggestion

  src = fetchFromGitLab {
```
Some of your ws formatting it a little idiosyncratic; rather than fixing manually, mind hosing it down with https://github.com/nix-community/nixpkgs-fmt ?

Reply:
![image](https://user-images.githubusercontent.com/6453661/89716647-739af880-d9b7-11ea-8ee7-05137c00bac9.png)
If makes it worse :/
nixfmt did a better job.